tools/api-markdown-documenter/src/api-item-transforms/TsdocNodeTransforms.ts

```diff
@@ -246,6 +257,35 @@ export function transformTsdocLinkTag(
 	);
 }
 
+/**
+ * Converts a {@link @microsoft/tsdoc#DocInlineTag} to a {@link SpanNode} (or `undefined` if the input is a `{@label}` tag).
+ *
+ * @remarks
+ * Custom inline tags are not something the system can do anything with inherently.
+ * In the future, we may be able to add extensibility points for transforming custom inline tags.
+ * But for now, we will simply emit them as plain text in the output.
```

Comment:
Not sure if it's worth adding here; up to you.
```suggestion
 * But for now, we will simply emit them as italicized plain text in the output.
```